graphql_compiler/tests/snapshot_tests/test_cost_estimation.py

```diff
@@ -1185,15 +1240,22 @@ def test_get_in_collection_filter_selectivity(self) -> None:
         self.assertEqual(expected_selectivity, selectivity)
 
         statistics_with_distinct_birthday_values_data = LocalStatistics(
-            dict(), distinct_birthday_values_data
+            dict(), distinct_field_values_counts=distinct_birthday_values_data
```

Comment:
Unrelated to this PR, but mypy found this issue so I had to fix the test.

Reply:
See above, I think this is caused by your API change on `LocalStatistics` that I point out above. This is a fine change, and I like kwarg-only arguments, so don't change it back :)